aclogview/Program.cs

```diff
@@ -5,6 +5,39 @@
 using System.Windows.Forms;
 
 namespace aclogview {
+    using Heading;   
+
+    public class Globals
+    {
+        private static bool _useHex;       
+
+        public static bool UseHex
+        {
+            get
+            {
+                // Reads are usually simple
+                return _useHex;
+            }
+            set
+            {
+                // You can add logic here for race conditions,
+                // or other measurements
+                _useHex = value;
+            }
+        }
+
+        public string FormatGuid(uint theValue)
+        {
+            if (Globals.UseHex)
+            {
+                return "0x" + theValue.ToString("X");
+            }
+
+            return theValue.ToString();
+        }
```

Comment:
This is a dupe of FormatGuid in Utility class. You should only have it in one spot.